README.md

```diff
@@ -100,18 +68,64 @@ Much of the philosophy behind Bedrock is inspired by the [Twelve-Factor App](htt
 - `WP_SITEURL` - Full URL to WordPress including subdirectory (https://example.com/wp)
 - `AUTH_KEY`, `SECURE_AUTH_KEY`, `LOGGED_IN_KEY`, `NONCE_KEY`, `AUTH_SALT`, `SECURE_AUTH_SALT`, `LOGGED_IN_SALT`, `NONCE_SALT`
   - Generate with [wp-cli-dotenv-command](https://github.com/aaemnnosttv/wp-cli-dotenv-command)
-  - Generate with [our WordPress salts generator](https://roots.io/salts.html)
+  - Generate with [Bedrock's WordPress salts generator](https://roots.io/salts.html)
```

Comment:
If Pantheon is already going to generate these keys and salts, maybe "Generate" could be changed to "Regenerate".